paparazzi/paparazzi/src/main/java/app/cash/paparazzi/accessibility/AccessibilityOverlayView.kt

```diff
@@ -32,9 +34,20 @@ internal class AccessibilityOverlayView(context: Context) : FrameLayout(context)
     style = Paint.Style.STROKE
   }
 
+  private lateinit var location: IntArray
+
   init {
     // Required for onDraw to be called
     setWillNotDraw(false)
+
+    // We can't get the location on screen until the view is attached.
+    addOnAttachStateChangeListener(object : OnAttachStateChangeListener {
```

Comment:
does this need to be detached in onViewDetachedFromWindow, lest we cause a memory leak? cc: @gamepro65

Reply:
yep good call